src/components/varsler/VarslerNy.tsx

```diff
@@ -0,0 +1,219 @@
+import { CheckmarkCircleFillIcon, ExclamationmarkTriangleFillIcon } from '@navikt/aksel-icons';
+import { Alert, HStack, Pagination, Table, VStack } from '@navikt/ds-react';
+import { type ReactNode, useState } from 'react';
+import { emptyReplacement, getVarselTekst } from 'src/app/personside/infotabs/varsel/varsel-utils';
+import VarselMeldinger from 'src/app/personside/infotabs/varsel/varselDetaljer/VarselMeldinger';
+import { useVarslerData } from 'src/lib/clients/modiapersonoversikt-api';
+import {
+    type DittNavEvent,
+    type FeiletVarsling,
+    type UnifiedVarsel,
+    type UnifiedVarsel as UnifiedVarselModell,
+    type Varsel as VarselModell,
+    type VarslerResult,
+    isDittNavEvent
+} from 'src/models/varsel';
+import { datoSynkende } from 'src/utils/date-utils';
+import { ENDASH, formaterDato } from 'src/utils/string-utils';
+
+function VarslerNy() {
+    const [page, setPage] = useState(1);
+    const rowsPerPage = 22;
+
+    const varslerResponse = useVarslerData();
```

Comment:
Du har vel ikke noen `<Suspense/>` under routen og over denne komponenten? Gjerne legg inn en ErrorBoundary og suspense boundary som håndterer feil og loading state for denne queryen.